src/backend/src/controllers/description-Bullets.controllers.ts

```diff
@@ -0,0 +1,36 @@
+import { Request, Response } from 'express';
+import { validationResult } from 'express-validator';
+import prisma from '../prisma/prisma';
+
+export const checkWorkPackageDescriptionBullet = async (req: Request, res: Response) => {
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    return res.status(400).json({ errors: errors.array() });
+  }
+
+  const { body } = req;
+  const { submitterId, descriptionBulletId } = body;
```

Comment:
a cleaner way to combine this and line 14 is to do `const { submitterId, descriptionBulletId: descriptionId }` but actually what's even easier is you can change the parameter name in the routes to be `descriptionId` in the first place